firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/FirebaseVertexAI.kt

```diff
@@ -130,9 +138,9 @@ internal constructor(
      */
     @JvmStatic
     @JvmOverloads
-    public fun getInstance(app: FirebaseApp = Firebase.app, location: String): FirebaseVertexAI {
+    public fun getInstance(app: FirebaseApp, location: String = "us-central1"): FirebaseVertexAI {
```

Comment:
This is a breaking change since the it's no longer possible to call `getInstance(location)`

Reply:
Unintentional change from some iteration I was doing, reverted.